### PR TITLE
Ui style

### PR DESCRIPTION
**Pull request by joe429273392**
renderer: Add metadata and basic structure
* App.js : add some metadata to the catalogsInfo Object
* Catalog.js : add the display of metadata
* Content.js : change the basic structure of content and display them
* ContentItem : add the structure and display of contentItem

### CODE REVIEW

File: renderer/src/App.js [hunk 1]
```diff
@@ -31,13 +31,19 @@ class App extends Component {
       name: item.name,
       commit: item.commit,
     }));
-
+    const catalogsInfo = {
+      'name': tuture.name,
+      'language': tuture.language,
+      'maintainer': tuture.maintainer,
+      'topics': tuture.topics,
+    };
```
Comment (pftom) [inline renderer/src/App.js]: Here you can use ES6 Destructuring: `const { a, b, c } = obj;`

At this time, you maybe need:

```javascript
  const { name, language, maintainer, topics } = tuture;
```

It's easy, uh! 😁

File: renderer/src/components/Catalog.js [hunk 2]
```diff
@@ -12,21 +12,41 @@ export default class Catalog extends Component {
...
-        defaultSelectedKeys={[selectKey]}
-        mode='inline'
-      >
-      {
-        catalogs.map((item, key) => (
-          <Menu.Item key={key}>{item.name}({item.commit})</Menu.Item>
-        ))
-      }
-      </Menu>
+      <div style={{display:'flex',flexDirection:'column', float:'left', height:'100%'}}>
```
Comment (pftom) [inline renderer/src/components/Catalog.js]: Here, your code-style isn's so good.

You be better write this:

```html
<div style={{ display: 'flex', flexDirection: 'column', float: 'left', height: '100%' }}></div>
```

File: renderer/src/components/Catalog.js [hunk 3]
```diff
@@ -12,21 +12,41 @@ export default class Catalog extends Component {
...
+          mode='inline'
+        >
+        {
+          catalogs.map((item, key) => (
+            <Menu.Item key={key}>{item.name}({item.commit})</Menu.Item>
+          ))
+        }
+              
+        </Menu>
+        <div style={{position:'absolute', bottom:'0', paddingLeft:'24px'}}>
```
Comment (pftom) [inline renderer/src/components/Catalog.js]: Here, the same as this code-style

File: renderer/src/components/ContentItem.js [hunk 4]
```diff
@@ -3,8 +3,13 @@ import React, { Component } from 'react';
 export default class ContentItem extends Component {
   render() {
     const { renderItem } = this.props;
+    console.log(renderItem);
     return (
-      <div>hhhh</div>
+      <div>
```
Comment (pftom) [inline renderer/src/components/ContentItem.js]: Here, use `Destructuring `, 

```javascript
  const { file, change, explain } = this.props.renderItem
 ```

Use this

File: renderer/src/components/Catalog.js [hunk 5]
```diff
@@ -12,21 +12,41 @@ export default class Catalog extends Component {
...
+          <hr/>
+          <p>name: {catalogsInfo.name}</p>
```
Comment (pftom) [inline renderer/src/components/Catalog.js]: Here, use `Destructuring `, 

```javascript
  const { file, change, explain } = this.props.renderItem
 ```

Use this

**Comment by mRcfps** [comment]
@pftom Please check these changes and merge this PR if it's done.

**Comment by pftom** [comment]
@mRcfps @joe429273392 I have finished review of this pull request

**Pull request merged**